src/fmu/dataio/_model/fields.py

```diff
@@ -103,7 +103,9 @@ class File(BaseModel):
     )
     """The path of a file relative to the case root."""
 
-    checksum_md5: Optional[str] = Field(examples=["kjhsdfvsdlfk23knerknvk23"])
+    checksum_md5: Optional[str] = Field(
+        default=None, examples=["kjhsdfvsdlfk23knerknvk23"]
+    )
```

Comment:
A test requires that validation cannot occur without checksum, still listed as Optional. Is this a backward compatibility thing patch-handled in Sumo?